src/System.Text.Json/tests/Serialization/Stream.WriteTests.cs

```diff
@@ -258,6 +259,488 @@ public static async Task LargeJsonFile(int bufferSize)
+                            FacebookAccountId = "10155043593360477",
+                            AllowNotes = 1,
+                            MediumMemberAt = 0,
+                            IsNsfw = false,
+                            IsWriterProgramEnrolled = true,
+                            IsQuarantined = false,
+                            Type = "User"
+                        }
+                    }, 
+                    Collaborators = new List<Collaborator>
+                    {
+                        new Collaborator
+                        {
+                            User  = new MentionedUser
+                            {
+                                UserId = "c4ccb3ab8d17",
+                                Name = "Ian Tinsley",
+                                Username = "itinsley",
+                                CreatedAt = 1438064675373,
+                                ImageId = string.Empty,
+                                BackgroundImageId = string.Empty,
+                                Bio = string.Empty,
+                                TwitterScreenName = string.Empty,
+                                FacebookAccountId = "10155043593360477",
+                                AllowNotes = 1,
+                                MediumMemberAt = 0,
+                                IsNsfw = false,
+                                IsWriterProgramEnrolled = true,
+                                IsQuarantined = false,
+                                Type = "User"
+                            },
+                            State = "visible"
+                        }
+                    },
+                    HideMeter = false,
+                    CollectionUserRelations = new List<CollectionUserRelation>
+                    {
+                        new CollectionUserRelation
+                        {
+                            CollectionId = "6b0b79fe3153",
+                            Role = "ADMIN",
+                            UserId = "78aeffcb483e"
+                        }
+                    },
+                    References = new References
+                    {
+                        User = new User
+                        {
+                            The7Ef192B7F545 = new MentionedUser
+                            {
+                                UserId = $"7ef192b7f545",
+                                Username = "fagnerbrack",
+                                Name = "Fagner Brack",
+                                CreatedAt = 1456628553936, // TODO should be datetime
+                                ImageId = $"{i}*h8Ph7pgNeQHZTiLlah3h-A.jpeg",
+                                BackgroundImageId = string.Empty,
+                                Bio = "I believe ideas should be open and free. This is a non-profit initiative to write about fundamentals you won't find anywhere else. ~7 min post every few weeks.",
+                                TwitterScreenName = "FagnerBrack",
```

Comment:
I don't think we should be doing things that attribute to real-world entities.  All of the things about users should be made up.